src/sentry/static/sentry/app/utils/requestError/requestError.tsx

```diff
@@ -1,5 +1,9 @@
 export default class RequestError extends Error {
-  constructor(method, path) {
+  responseJSON?: any;
```

Comment:
can it be a `object`?

Reply:
@EvanPurkhiser  I put it as `any` primarily because of https://github.com/DefinitelyTyped/DefinitelyTyped/blob/8a1b68be3a64e5d2aa1070f68cc935d668a976ad/types/jquery/v2/index.d.ts#L209

------

I can understand why we can assume it to be `object`, but I dont think we can refine this type to `object` (note the lowercase o); and I think it would be incorrect to do so.

There's some gotchas when we do this though.

The `object` type represents the non-primitive type (see https://www.typescriptlang.org/docs/handbook/basic-types.html#object). Essentially any type that is not the primitive type. 

In terms of code, something like this would be incorrect (even tho TS will still compile it):


```ts
const foo: object = JSON.parse('true');
```